pyramid_heroku/tests/test_client_addr.py

```diff
@@ -45,3 +46,27 @@ def test_spaces(self):
         ClientAddr(self.handler, self.registry)(self.request)
         self.handler.assert_called_with(self.request)
         self.assertEqual(self.request.client_addr, "1.2.3.4")
+
+    def test_app_behind_proxy(self):
+        from pyramid_heroku.client_addr import ClientAddr
+
+        registry = mock.Mock()
+        registry.settings = {"pyramid_heroku.reverse_proxy": True}
+        self.request.environ["REMOTE_ADDR"] = "127.0.0.1"  # load balancer
+        self.request.headers["X-Forwarded-For"] = "6.6.6.6,1.2.3.4,7.7.7.7"
+
+        ClientAddr(self.handler, registry)(self.request)
+        self.handler.assert_called_with(self.request)
+        self.assertEqual(self.request.client_addr, "1.2.3.4")
+
+    def test_app_reverse_proxy_wrong_configuration(self):
+        from pyramid_heroku.client_addr import ClientAddr
+
+        registry = mock.Mock()
+        registry.settings = {"pyramid_heroku.reverse_proxy": True}
+        self.request.environ["REMOTE_ADDR"] = "127.0.0.1"  # load balancer
+        self.request.headers["X-Forwarded-For"] = "1.2.3.4"
+
+        ClientAddr(self.handler, registry)(self.request)
+        self.handler.assert_called_with(self.request)
+        self.assertEqual(self.request.client_addr, "1.2.3.4")
```

Comment:
If `pyramid_heroku.reverse_proxy=true` but the app isn't behind the proxy it takes the last IP. 

This gives some room for spoofing IP if someone forgets to turn off this option or if the client somehow access the app directly.